### ISSUE TITLE
Release to all destinations

### ISSUE THREAD

**Issue by mschmidt-healthx** [microsoft/fastlane-plugin-appcenter#132]
The new appcenter_fetch_devices action supports getting devices for all destinations for an app using destinations="*"; it would be nice if the appcenter_upload action supported releasing to all destinations as well.

**Comment by benkane** [comment]
Good call. That sort of works right now, but only if you use the fetch devices action first. The logic for that as it lives in the fetch devices action is pretty straightforward is [here](https://github.com/microsoft/fastlane-plugin-appcenter/blob/master/lib/fastlane/plugin/appcenter/actions/appcenter_fetch_devices_action.rb#L26). It could be extracted further to the helper for both actions to use.

**Comment by benkane** [comment]
As a side note it'd be nice if the API itself supported this so we didn't need to do it client side. Not sure where to leave that feedback. Maybe someone can pass it along to the API team

**Comment by JanGorman** [comment]
Interesting, I was about to post something similar. We did however also noticed that the comma separated list isn't working as expected. I took a quick look at the [implementation](https://github.com/microsoft/fastlane-plugin-appcenter/blob/master/lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb#L177) which looks to be doing the right thing but in our case, the app was only released to one of the groups. In code we have: `destinations: 'Lounge BI,Lounge Mobile,Public Testers',` – It is only released to the the last group `Public Testers`:

![Screenshot 2019-10-18 at 09 22 53](https://user-images.githubusercontent.com/138166/67100965-fd779c00-f1c0-11e9-8c32-4832bcc173fc.png)
